pkgs/applications/display-managers/ly/default.nix

```diff
@@ -1,24 +1,29 @@
-{ stdenv, lib, fetchFromGitHub, git, linux-pam, libxcb }:
+{ stdenv
+, lib
+, fetchFromGitHub
+, linux-pam
+, libxcb
+, makeBinaryWrapper
+, zig_0_12
+, callPackage
+}:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
   pname = "ly";
-  version = "0.6.0";
+  version = "1.0.0";
 
   src = fetchFromGitHub {
     owner = "fairyglade";
     repo = "ly";
-    rev = "v${version}";
-    hash = "sha256-78XD6DK9aQi8hITWJWnFZ3U9zWTcuw3vtRiU3Lhu7O4=";
-    fetchSubmodules = true;
+    rev = "v1.0.0";
+    hash = "sha256-IwZ9QWVrQz/DIIcR4XOW5q54gYDFn5prmujnS3sSquc=";
   };
 
```

Comment:
Expose `nixosTests.ly` in `passthru.tests`

Reply:
CC @Vonfry.